src/DurableTask.Core/Logging/LogEvents.cs

```diff
@@ -632,7 +632,7 @@ public TerminatingInstance(OrchestrationInstance instance, string reason)
                 StructuredEventSource.Log.TerminatingInstance(
                     this.InstanceId,
                     this.ExecutionId,
-                    this.Details,
+                    Details: string.Empty, // User-provided details may contain sensitive data, so we don't log it.
```

Comment:
Totally optional, but if you want to provide a bit more information, you could perhaps log the number of characters of data that was associated with this. For example:
```suggestion
                    Details: $"(Redacted {this.Details?.Length ?? 0} characters}), // User-provided details may contain sensitive data, so we don't log it.
```
The reason to *consider* this is if you want to try and distinguish between empty and non-empty, or perhaps to get a hint about whether there might be some large messages at play. It also could help make it clear that something was explicitly redacted. I don't have a concrete scenario for this, but it could be useful.

Reply:
Good idea, will incorporate.